### PR TITLE
Fix various issues

### LINKED ISSUE #1415: `bin/create-or-reset-admin` fails silently when password doesn't meet requirements
I can't create admin, the below command not working:
`docker compose run --rm firezone bin/create-or-reset-admin`

I have created proper .env file with
ADMIN_EMAIL
DEFAULT_ADMIN_PASSWORD
also tried 
RESET_ADMIN_ON_BOOT=true

and not working too.

Table users in db is empty:
image

### LINKED ISSUE #1458: Upgrade from 0.7.13 -> 0.7.16 - Invalid DNS server IP
**Describe the bug**
After upgrading from 0.7.13 -> 0.7.16, I get an error with the set of IPv6 DNS servers I have configured for clients (see logs below).

**To Reproduce**
`firezone.rb` has this configuration:

```ruby
default['firezone']['wireguard']['dns'] = '1.1.1.1, 1.0.0.1, 8.8.8.8, 8.8.4.4, 2606:4700:4700::1111, 2606:4700:4700::1001, 2001:4860:4860::8888, 2001:4860:4860::8844'
```

Removing the addresses from `firezone.rb` does not fix; rather they are set in the DB.

**Expected behavior**
The IPv6 DNS servers are allowed

**Screenshots / Logs**

```
2023-02-23_07:18:39.15981     ** (EXIT) an exception was raised:
2023-02-23_07:18:39.15981         ** (RuntimeError) Found 1 configuration errors:
2023-02-23_07:18:39.15982
2023-02-23_07:18:39.15982
2023-02-23_07:18:39.15982 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2023-02-23_07:18:39.15982
2023-02-23_07:18:39.15983 Invalid configuration for 'default_client_dns' retrieved from database configuration default_client_dns.
2023-02-23_07:18:39.15983
2023-02-23_07:18:39.15983 Errors:
2023-02-23_07:18:39.15983
2023-02-23_07:18:39.15983  - `" 2001:4860:4860::8844"`: must be one of: Elixir.FzHttp.Types.IP, string, 2001:4860:4860::8844 is not a valid FQDN
2023-02-23_07:18:39.15984  - `" 2001:4860:4860::8888"`: must be one of: Elixir.FzHttp.Types.IP, string, 2001:4860:4860::8888 is not a valid FQDN
2023-02-23_07:18:39.15984  - `" 2606:4700:4700::1001"`: must be one of: Elixir.FzHttp.Types.IP, string, 2606:4700:4700::1001 is not a valid FQDN
2023-02-23_07:18:39.15984  - `" 2606:4700:4700::1111"`: must be one of: Elixir.FzHttp.Types.IP, string, 2606:4700:4700::1111 is not a valid FQDN
2023-02-23_07:18:39.15985             (fz_http 0.7.16) lib/fz_http/config/errors.ex:10: FzHttp.Config.Errors.raise_error!/1
2023-02-23_07:18:39.15987             (fz_http 0.7.16) lib/fz_http/application.ex:12: FzHttp.Application.start/2
2023-02-23_07:18:39.15987             (kernel 8.5) application_master.erl:293: :application_master.start_it_old/4
```

**Platform (please complete the following information):**
 - Firezone Version 0.7.16
 - OS: Linux
 - Distro: Ubuntu 22.04
 - Kernel version: 5.15
 - Deployment method: Omnibus

**Additional context**
N/A

### LINKED ISSUE #1460: Issues when adding new device
**Describe the bug**
After upgrading from 0.7.13 to 0.7.15 (and then 0.7.16 too), it seems that when you try to add a new device on the WebUI nothing happens and the WireGuard configuration isn't prompted.
We have two Firezone instances, one running on Ubuntu 22.04 and one running on Ubuntu 20.04 and we have deployed Firezone with Docker.
Both instances have the same problem.

**To Reproduce**
Go on the Firezone WebUI and try to add a new device.

### LINKED ISSUE #1461: `/var/firezone/private_key` should not be required to exist on boot
image

It is generated on first boot here: https://github.com/firezone/firezone/blob/master/apps/fz_vpn/lib/fz_vpn/keypair.ex#L6

### PR DESCRIPTION
**Pull request by AndrewDryga**
Closes #1461
Closes #1458
Closes #1415
Closes #1460

### CODE REVIEW

File: rel/overlays/bin/server [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+set -e
```
Comment (AndrewDryga) [inline rel/overlays/bin/server:2]: We really don't want to boot firezone if migrations or admin creation fails

File: config/config.exs [hunk 2]
```diff
@@ -44,11 +44,19 @@ config :fz_http, FzHttpWeb.Endpoint,
 
 config :fz_http,
   wireguard_ipv4_enabled: true,
-  wireguard_ipv4_network: "100.64.0.0/10",
-  wireguard_ipv4_address: "100.64.0.1",
+  wireguard_ipv4_network: %{__struct__: Postgrex.INET, address: {100, 64, 0, 0}, netmask: 10},
```
Comment (AndrewDryga) [inline config/config.exs:47]: This is intentional: struct is not available when Elixir reads the config, so we create it using `__struct__` and module name.

File: rel/overlays/bin/server (hunk 1 above)
Comment (jamilbk) [reply rel/overlays/bin/server:2]: Agree

**Review by jamilbk — APPROVED** [review]
Amazing quick fixes, thanks!

File: config/config.exs (hunk 2 above)
Comment (jamilbk) [reply config/config.exs:47]: I figured. Nice workaround!

**Pull request merged**